src/hotspot/share/runtime/arguments.cpp

```diff
@@ -1773,9 +1773,15 @@ unsigned int addexports_count = 0;
 unsigned int addopens_count = 0;
 unsigned int patch_mod_count = 0;
 unsigned int enable_native_access_count = 0;
+bool patch_mod_javabase = false;
 
 // Check the consistency of vm_init_args
 bool Arguments::check_vm_args_consistency() {
+  // This may modify compiler flags. Must be called before CompilerConfig::check_args_consistency()
+  if (!CDSConfig::check_vm_args_consistency(patch_mod_javabase, mode_flag_cmd_line)) {
+    return false;
+  }
```

Comment:
`check_vm_args_consistency` is only directly referenced in this file now. Can it be declared as local `static`?

Reply:
`Arguments::check_vm_args_consistency()` is also called from jvmFlag.cpp so we can't make it local to arguments.cpp.

Reply:
Sorry, I copy/past wrong name. I meant `patch_mod_javabase` variable. It can be local static I think.

Reply:
Fixed.